test/integration/test-extensions-loading.js

```diff
@@ -73,14 +73,20 @@ describe('test extensions loading in multiple orders', function () {
     );
   });
 
-  it('one extension, extension loads first, all scripts in footer', () => {
+  // TODO(#30528): skip this test as it doesn't make sense. The script
```

Comment:
How about deleting these tests instead of skipping, since this will never be a valid use case?

Reply:
@zhouyx wanted to confirm with you on this one. This should be OK to remove both footer tests correct? It's just that it would never be valid amp in the first place to have the js resources on the footer

Reply:
The tests were requested by #3228. By then the idea was to test that an extension will be upgraded correctly despite the JS load order. 

Placing the script in footer was to mock loading order.

Reply:
@zhouyx do you think there is any other way to test it? we're trying to make our postHTML transforms be as close in behavior to the google amp cache transformers. in cases for module/nomodule for example we wouldn't be able to do a transform for these scripts

Reply:
im still going to go ahead and skip this for now since getting the module tests in is higher priority, but i'd like to tackle this load order issue afterwards

Reply:
Sorry about the delay. 

does we apply transformer change to integration test as well? If so I agree this may not be a good test case. 

I wonder if it is possible to add some delay to the test server when a script is being requested? In that case we don't need to change the html.